packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -80,6 +80,7 @@ class CupertinoScrollbar extends RawScrollbar {
     this.thicknessWhileDragging = defaultThicknessWhileDragging,
     Radius radius = defaultRadius,
     this.radiusWhileDragging = defaultRadiusWhileDragging,
+    this.color,
```

Comment:
If we were to add this, it would probably be better to override the super class' `RawScrollbar.thumbColor`.

Reply:
Yes, I forgot to add it in the super
`thumbColor: color,`